changelog.txt

```diff
@@ -8,11 +8,12 @@ V.Next
 - [PATCH] Fix accidental code change that disabled PoP for auth code grant flow (#1661)
 - [MINOR] Add flighting parameters to commmandParameters (#1562)
 - [MINOR] Add ropc command and ropc flow to BaseController (#1539)
+- [PATCH] Ensure a device pop manager is provided when PoPAuthenticationScheme is requested of the broker (#1814728)
```

Comment:
What does this number mean? (#1814728)

This number generally represents the pull request id however, that doesn't seem to be the case here

Reply:
Sorry, I was using the bug id... which I thought made more sense.  Sorry about that.